src/engine/channelmixer.cpp

```diff
@@ -69,7 +69,7 @@ void ChannelMixer::applyEffectsInPlaceAndMixChannels(
     //    B) Applies effects to the buffer, modifying the original input buffer
     // 4. Mix the channel buffers together to make pOutput, overwriting the pOutput buffer from the last engine callback
     ScopedTimer t(QStringLiteral("EngineMixer::applyEffectsInPlaceAndMixChannels"));
-    SampleUtil::clear(pOutput, iBufferSize);
+    SampleUtil::clear(pOutput, bufferSize);
```

Comment:
I think this needs a `std::size_t` overload, wdyt?

Reply:
An overload is ambiguous, we would need two function names and adjust all the callers accordingly. This seems to much work to me.

Reply:
well, its only ambiguous if an implicit conversion was required. That shouldn't be in many places. I will experiment in a follow up.